wa/commands/runstatus.py

```diff
@@ -0,0 +1,298 @@
+                  )
+        header += "UUID: {}\n".format(info.uuid)
+        if info.run_name:
+            header += "Run name: {}\n".format(info.run_name)
+        if info.project:
+            header += "Project: {}\n".format(info.project)
+        if info.project_stage:
+            header += "Project stage: {}\n".format(info.project_stage)
+
+        if info.start_time:
+            duration = _seconds_as_smh(self.elapsed_time.total_seconds())
+            header += ("Start time: {}\n"
+                       "Duration: {:02}:{:02}:{:02}\n"
+                       ).format(info.start_time,
+                                duration[2], duration[1], duration[0],
+                                )
+            if len(self.segmented['finished']) and not info.end_time:
+                p_duration = _seconds_as_smh(self.projected_duration.total_seconds())
+                header += "Projected time remaining: {:02}:{:02}:{:02}\n".format(
+                    p_duration[2], p_duration[1], p_duration[0]
+                )
+
+            elif self.ro.info.end_time:
+                header += f"End time: {info.end_time}\n"
+
+        return header
+
+    def generate_job_summary(self):
+        total = len(self.jobs)
+        num_fin = len(self.segmented['finished'])
+
+        summary = ('\n=========================\n'
+                   'Job Summary'
+                   '\n=========================\n\n'
+                   )
+        summary += f'Total: {total}, Completed: {num_fin} ({(num_fin/total)*100}%)\n'
+
+        job_list = []
+        for run_state, jobs in ((k, v) for k, v in self.segmented.items() if v):
+            if run_state == 'finished':
+                end_states = {
+                                Status.PARTIAL: 0, Status.FAILED: 0,
+                                Status.ABORTED: 0, Status.OK: 0,
+                                Status.SKIPPED: 0
+                              }
+                for job in jobs:
+                    end_states[job.status] += 1
+
+                for status, count in end_states.items():
+                    if count:
+                        job_list.append('{} {}'.format(
+                            count,
+                            self.formatter.highlight_keyword(status.name.lower())
+                            )
+                        )
+
+            else:
+                count = len(jobs)
+                job_list.append('{} {}'.format(
+                    count,
+                    self.formatter.highlight_keyword(run_state)
+                    )
+                )
+
+        summary = summary + ', '.join(job_list) + '\n'
+        return summary
+
+    def generate_job_detail(self):
+        detail = ('\n=========================\n'
+                  'Job Detail'
+                  '\n=========================\n\n'
+                  )
+
+        for segment, jobstates in self.segmented.items():
+            if jobstates:
```

Comment:
I would switch this up to 
```
if not jobstates:
   continue
for job in jobstates:
   # ...
```
This will allow you to remove an indent level. Generally, the fewer indent levels there are in the code, the better.